cmd/cli/node/list.go

```diff
@@ -69,15 +73,22 @@ func (o *ListOptions) run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	if o.FilterByApproval != "" {
+		if !slices.Contains(filterApprovalValues, o.FilterByApproval) {
+			return fmt.Errorf("cannot use '%s' as filter approval value, should be one of: %q", o.FilterByApproval, filterApprovalValues)
+		}
+	}
+
 	if o.FilterByStatus != "" {
 		if !slices.Contains(filterStatusValues, o.FilterByStatus) {
-			util.Fatal(cmd, fmt.Errorf("cannot use '%s' as filter status value, should be one of: %q", o.FilterByStatus, filterStatusValues), 1)
+			return fmt.Errorf("cannot use '%s' as filter status value, should be one of: %q", o.FilterByStatus, filterStatusValues)
```

Comment:
consider printing the specific flag name in the error message e.g. `filter-status` and `filter-approval`